tests/scanners/zap/test_setup.py

```diff
@@ -236,6 +236,51 @@ def test_setup_include_urls(test_config):
     assert "def" in find_context(test_zap.automation_config)["includePaths"]
 
 
+def test_setup_replacer(test_config):
+    # test if deleteAllRules is set to True by default when it is not set
+    test_zap = ZapNone(config=test_config)
+    test_zap.setup()
+
+    for item in test_zap.automation_config["jobs"]:
+        if item["type"] == "replacer":
```

Comment:
this looks like it will always be false, unless the test_config is modified, so the following assertion will not be evaluated

Reply:
nice catch. Thanks.